torch/csrc/jit/passes/symbolic_shape_analysis.cpp

```diff
@@ -198,10 +195,12 @@ struct SymbolicShapeAnalyzer {
             if (!norm_index) {
               continue;
             }
-            if (tensor_shape[*norm_index].is_static() ||
-                substitute_symbolic_dims) {
+            if (tensor_shape[*norm_index].is_static()) {
```

Comment:
It's a comment on already existing code, but still: is my understanding correct that `node_` here corresponds to a node in the real graph on which we run the shape prop pass, while the `graph_` is the shape-compute graph? If so, can we reflect in their name that they are explicitly separate? It would help to comprehend the code I think. Otherwise I keep catching myself assuming that `graph_` must be the same as `node_->owningGraph()`, which I think is wrong.

Reply:
yes, that is correct. what would you suggest, naming-wise?

Reply:
Maybe explicitly call it "shape_function_graph_" or something like that?